lib/cli/devMode.js

```diff
@@ -12,11 +12,29 @@ const chokidar = require("chokidar");
  * @param {String} handle - template handle
  * @param {String} testName - Test name (empty string to run all tests)
  * @param {boolean} renderInput - Open browser and show the HTML from input view
+ * @param {String} templateType - Template type (reconciliationText, accountTemplate)
  */
-async function watchLiquidTest(firmId, handle, testName, renderInput) {
-  consola.info(`Watching for changes related to reconciliation "${handle}" to run a new test...`);
+async function watchLiquidTest(firmId, handle, testName, renderInput, templateType) {
+  if (templateType !== "reconciliationText" && templateType !== "accountTemplate") {
+    consola.error(`Template type is missing or invalid`);
+    process.exit(1);
+  }
+
+  let filePath;
+  switch (templateType) {
+    case "reconciliationText":
+      consola.info(`Watching for changes related to reconciliation "${handle}" to run a new test...`);
+      filePath = path.resolve(process.cwd(), "reconciliation_texts", handle, "tests", `${handle}_liquid_test.yml`);
+      break;
+    case "accountTemplate":
+      consola.info(`Watching for changes related to account template "${handle}" to run a new test...`);
+      filePath = path.resolve(process.cwd(), "account_templates", handle, "tests", `${handle}_liquid_test.yml`);
+      break;
+    default:
+      break;
+  }
```

Comment:
This could be taken from fsUtils actually, right?

something like `fsUtils.FOLDERS[templateType]` may work :thinking: